sdk/appcenter/src/main/java/com/microsoft/appcenter/http/HttpClientNetworkStateHandler.java

```diff
@@ -67,82 +64,34 @@ public void reopen() {
 
     @Override
     public synchronized void onNetworkStateUpdated(boolean connected) {
-        if (connected) {
+        if (connected && mCalls.size() > 0) {
```

Comment:
Can you log "Network is available" when it is connected? Split below the log message or you can print below log message even though `mCalls.size() == 0`.

Reply:
According to me, it's better now. When you have multiple apps in background it's always spamming logs otherwise.

Reply:
Network availability is logged in `NetworkStateHelper`